README.md

```diff
@@ -142,7 +159,7 @@ If a client is from the same country as we are, the invoice we give him has a no
 
 ### So that we know how much you should be paid.
 
-We all have hourly rates.
+We all work on hourly contracts.
```

Comment:
I believe that you mean `hourly fees` – as in an income received in an hourly manner, due to agreement.
```suggestion
We all work on hourly fees.
```